shared/people/task/container.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+import {Task} from '.'
+import * as PeopleGen from '../../actions/people-gen'
+import * as Types from '../../constants/types/people'
+import * as Tabs from '../../constants/tabs'
+import * as SettingsTabs from '../../constants/settings'
+import {connect, branch, compose, renderNothing} from '../../util/container'
+import {type TypedState} from '../../constants/reducer'
+import {createGetMyProfile} from '../../actions/tracker-gen'
+import {navigateAppend, switchTo, navigateTo} from '../../actions/route-tree'
+import {createShowUserProfile} from '../../actions/profile-gen'
+import openURL from '../../util/open-url'
+import {isMobile} from '../../constants/platform'
+
+const installLinkURL = 'https://keybase.io/download'
+
+const onSkipTodo = (type: Types.TodoType, dispatch: Dispatch) => () =>
+  dispatch(PeopleGen.createSkipTodo({type}))
+
+const mapStateToProps = (state: TypedState) => ({myUsername: state.config.username})
+
+const makeConnector = (mapDispatchToProps, mapStateToProps?, mergeProps?) =>
+  mergeProps
+    ? connect(mapStateToProps || (() => ({})), mapDispatchToProps, mergeProps)
+    : connect(mapStateToProps || (() => ({})), mapDispatchToProps)
+
+// ----- BIO ----- //
+const bioConnector = makeConnector(
```

Comment:
why not just call connect here, makeConnector doesn't really save you any space or clarity